src/dev/build/tasks/build_kibana_example_plugins.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import Path from 'path';
+import Fs from 'fs';
+import { REPO_ROOT } from '@kbn/utils';
+import { exec, mkdirp, copyAll, Task } from '../lib';
+
+export const BuildKibanaExamplePlugins: Task = {
+  description: 'Building distributable versions of Kibana example plugins',
+  async run(config, log, build) {
+    const examplesDir = Path.resolve(REPO_ROOT, 'examples');
+    const args = [
+      '../../scripts/plugin_helpers',
+      'build',
+      `--kibana-version=${config.getBuildVersion()}`,
+    ];
+
+    // causes build errors
+    const exclude = [
```

Comment:
We're looking into https://github.com/elastic/kibana/pull/104512 as a potential fix here.

Reply:
#104512 is merged, part 2 to fix this call is at #104556

Reply:
#104556 is merged